packages/reports/addon/components/report-builder/sidebar.ts

```diff
@@ -143,10 +144,11 @@ export default class ReportBuilderSidebar extends Component<Args> {
       return yield [];
     }
 
-    return yield dataSource.namespaces.map(({ name, displayName, description }) => {
+    return yield dataSource.namespaces.map(({ name, displayName, description, hide }) => {
```

Comment:
Instead of filtering the data source in the template, could we filter out the data sources at a lower level?